scripts/python/read_dense.py

```diff
@@ -50,17 +55,42 @@ def read_array(path):
     return np.transpose(array, (1, 0, 2)).squeeze()
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-d", "--depth_map",
+                        help="path to depth map", type=str, required=True)
+    parser.add_argument("-n", "--normal_map",
+                        help="path to normal map", type=str, required=True)
+    parser.add_argument("--min_depth_percentile",
+                        help="minimum visualization depth percentile",
+                        type=float, default=5)
+    parser.add_argument("--max_depth_percentile",
+                        help="maximum visualization depth percentile",
+                        type=float, default=95)
+    args = parser.parse_args()
+    return args
+
+
 def main():
+    args = parse_args()
+
+    if args.min_depth > args.max_depth:
+        raise ValueError(
+            "Minimum depth should be less than or equal to the maximum depth.")
+
     # Read depth and normal maps corresponding to the same image.
-    depth_map = read_array(
-        "path/to/dense/stereo/depth_maps/image1.jpg.photometric.bin")
-    normal_map = read_array(
-        "path/to/dense/stereo/normal_maps/image1.jpg.photometric.bin")
+    if not os.path.exists(args.depth_map):
+        raise FileNotFoundError("File not found: {}".format(args.depth_map))
+
+    if not os.path.exists(args.normal_map):
+        raise FileNotFoundError("File not found: {}".format(args.normal_map))
 
-    import pylab as plt
+    depth_map = read_array(args.depth_map)
+    normal_map = read_array(args.normal_map)
```

Comment:
Currently, the normal map is read, but not being used. I would either display it together with the depth or remove this line. What do you think?

Reply:
Agreed.

Reply:
I would add a visualization of the normal map, just to show that it can be loaded using the same function as well.